kafka-broker/kafka-broker-install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+## Script to install a kafka & zookeeper broker
+
+# Install kafka & zookeeper
+	sudo useradd kafka -m
```

Comment:
There's no need for any of these lines to be indented

Reply:
fixed as requested